lib/utils.pm

```diff
@@ -3154,4 +3156,32 @@ sub enable_console_kernel_log {
     record_info("Content of /proc/sys/kernel/printk", script_output("cat /proc/sys/kernel/printk"));
 }
 
+=head2 is_disk_image
+
+ is_disk_image;
+
+Identify whether test runs with linux disk image built by kiwi or similar programs.
+HDD_1 is usually used if disk image is available on openQA server. Test run attempts
+downloading HDD_1, failure of which leads to failed test run. INSTALL_HDD_IMAGE is
+introduced for installation with disk image which might be located somewhere else
+and also more flexibility.
+=cut
+
+sub is_disk_image {
+    return 1 if ((get_var('HDD_1') or get_var('INSTALL_HDD_IMAGE')) and get_var('BOOT_HDD_IMAGE'));
```

Comment:
If you have `HDD_1` and `BOOT_HDD_IMAGE` you will always end in installation. Logic should be `(get_var('HDD_1')  && get_var('INSTALL_HDD_IMAGE')` to detect installation scenario.